package.json

```diff
@@ -13,16 +13,15 @@
         "start:server": "tinylicious"
     },
     "dependencies": {
-        "@fluid-experimental/data-objects": "^0.36.0",
-        "@fluid-experimental/fluid-static": "^0.36.0",
-        "@fluid-experimental/get-container": "^0.36.0",
-        "react": "^17.0.1",
-        "react-dom": "^17.0.1",
+        "@fluid-experimental/tinylicious-client": "^0.39.3",
+        "@fluidframework/map": "^0.39.3",
+        "react": "^17.0.2",
+        "react-dom": "^17.0.2",
         "vue": "^3.0.5"
     },
     "devDependencies": {
         "@types/react": "^17.0.2",
-        "@types/react-dom": "^17.0.1",
+        "@types/react-dom": "^17.0.2",
         "clean-webpack-plugin": "^3.0.0",
         "concurrently": "^5.3.0",
```

Comment:
We should bump the version of tinylicious here to be 0.4.21640 as it will get used by `npm run start:server` and the ports wont match up with default port used by the tinylicious-driver 0.39.3 that tinylicious-client depends on